src/XmHTML/lib/Motif/motif.c

```diff
@@ -281,7 +281,7 @@ DestroyImage(XImage *image)
 }
 
 static unsigned long
-GetPixelWrapper(XImage *ximage, Dimension x, Dimension y)
+GetPixelWrapper(XImage *ximage, _XtDimension  x, _XtDimension  y)
 {
```

Comment:
It might not matter, but I would expect _XtPosition to be more a more appropriate type for x and y.

Reply:
That might be. I used `_XtDimension` because that is the default prototype used in tka.h for the GetPixel function, but this is confusing because the original Dimension is an `unsigned int`  defined in miniparse.h. In Xutil.h (part of the X11 library, provided on the Mac via XQuartz), XGetPixel uses either `int` directly or the x and y are defined within a macro that calls get_pixel (Xlib.h), which uses `int` for x and y.

Intrinsic.h in the X11 library specifies _XtDimension as either `Dimension` (defined as `unsigned int`) or `int` depending on whether it is being used with a "wide" variable definition. They also say not to use _XtDimension and similar variable types in client code - not sure if that means XmHTML code. 

So either `_XtDimension`, `int` or `unsigned int` seem to be reasonable choices.

Reply:
Cool, given the prototype, _XtDimension does seem good.